aten/src/ATen/native/Indexing.cpp

```diff
@@ -250,4 +250,49 @@ Tensor & index_put_(Tensor & self, TensorList indices, const Tensor & value) {
   return src.put_(linearIndex, expandedValue);
 }
 
+Tensor & index_copy_(Tensor & self, int64_t dim, const Tensor & index, const Tensor & source) {
+  dim = maybe_wrap_dim(dim, self.dim());
+
+  if (index.dim() >= 2) {
+    runtime_error(
```

Comment:
should we `throw std::runtime_error` to be consistent with the code below?

Reply:
It's probably easier to change the std::runtime_error below to a call to `runtime_error` (which calls std::runtime_error). I'll change that.

Reply:
Nevermind, I remember the reason why I wrote it like this. The `runtime_error` here is cleaner because it takes a printf-style string, and the code below throws `std::runtime_error` because the string stream creates a string of some sort (which is not the `char*` `runtime_error` expects as its first argument.